src/sentry/api/endpoints/source_map_debug_blue_thunder_edition.py

```diff
@@ -275,143 +281,183 @@ def get_source_file_data(abs_path, project, release, event):
                 sourcemap_header = (
                     force_bytes(sourcemap_header) if sourcemap_header is not None else None
                 )
-
             try:
                 source_map_reference = find_sourcemap(
                     sourcemap_header, possible_release_file.file.getfile().read()
                 )
-                if source_map_reference is not None:
-                    source_map_reference = force_str(source_map_reference)
+                source_map_reference = (
+                    force_str(source_map_reference) if source_map_reference is not None else None
+                )
             except AssertionError:
                 pass
 
-            matching_source_map_name = None
-            if source_map_reference is not None:
-                if source_map_reference.startswith("data:"):
-                    source_map_reference = "Inline Sourcemap"
-                    path_data["source_map_lookup_result"] = "found"
-                else:
-                    matching_source_map_name = get_matching_source_map_location(
-                        possible_release_file.name, source_map_reference
-                    )
-
-            if matching_source_map_name is not None:
-                path_data["source_map_lookup_result"] = get_release_files_status_by_url(
-                    event, project, release, [matching_source_map_name]
-                )
-
-            return {
-                "matching_source_file_names": filenme_choices,
-                "matching_source_map_name": matching_source_map_name,
-                "source_map_reference": source_map_reference,
-                "source_file_lookup_result": "found",
-                "source_map_lookup_result": path_data["source_map_lookup_result"],
-            }
-
-    possible_release_artifact_bundles = ReleaseArtifactBundle.objects.filter(
-        organization_id=project.organization.id,
-        release_name=release.version,
-        artifact_bundle__projectartifactbundle__project_id=project.id,
-        artifact_bundle__artifactbundleindex__organization_id=project.organization.id,
-        artifact_bundle__artifactbundleindex__url__in=filenme_choices,
-    )
-    if len(possible_release_artifact_bundles) > 0:
-        path_data["source_file_lookup_result"] = (
-            "wrong-dist"
-            if path_data["source_file_lookup_result"] == "unsuccessful"
-            else path_data["source_file_lookup_result"]
+    # Look for source file in artifact indexes
+    if source_file_lookup_result != "found":
```

Comment:
Can we early return instead of having this boolean condition here?

Reply:
I think this might be implicitly resolved.